extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-sidebar-0.1.css';
+import {Layout} from '../../../src/layout';
+import {isExperimentOn} from '../../../src/experiments';
+
+
+/** @const */
+const EXPERIMENT = 'amp-sidebar';
+
+/** @const */
+const TAG = 'AmpSidebar';
+
+export class AmpSidebar extends AMP.BaseElement {
+  /** @override */
+  isLayoutSupported(layout) {
+    return layout == Layout.NOLAYOUT;
```

Comment:
FYI: This is especially good because we make it clear that the element currently must start hidden. If tomorrow we change this - we will be able to do so naturally and we'll know we are not breaking the existing code.